hydrus/core/HydrusKritaHandling.py

```diff
@@ -28,45 +32,50 @@ def ExtractZippedImageToPath( path_to_zip, temp_path_file ):
         
         raise HydrusExceptions.NoThumbnailFileException( f'This krita file had no {KRITA_FILE_MERGED} or {KRITA_FILE_THUMB}!' )
         
+
+def MergedPILImageFromKRA(path):
+
+    try:
+
+        file_obj = io.BytesIO( HydrusArchiveHandling.ReadSingleFileFromZip( path, KRITA_FILE_MERGED ))
```

Comment:
This seems inefficient since you're reading the whole file into memory and then turning it back into a bytes stream. 

I _think_ this would be better:
```py
file_obj = HydrusArchiveHandling.GetZipAsPath( path, KRITA_FILE_MERGED ).open('rb')
```